pkg/apiserver/rest/dualwriter_test.go

```diff
@@ -14,6 +14,13 @@ import (
 	"k8s.io/apiserver/pkg/endpoints/request"
 )
 
+var now = time.Now()
+
```

Comment:
these are moved from `storage_mocks_test.go` -- a file that did not contain any tests